pkg/cmd/pr/pr.go

```diff
@@ -95,6 +95,7 @@ func NewCmdPullRequest() (*cobra.Command, *Options) {
 	eo := &o.EnvironmentPullRequestOptions
 	cmd.Flags().StringVarP(&eo.CommitTitle, "commit-title", "", "", "the commit title")
 	cmd.Flags().StringVarP(&eo.CommitMessage, "commit-message", "", "", "the commit message")
+	cmd.Flags().StringVarP(&eo.BaseBranchName, "base-branch-name", "b", "", "the base branch name to use for new pull requests")
```

Comment:
Is this used in the code somwhere? I dont see it ...

Reply:
@ankitm123 It is used here: https://github.com/jenkins-x-plugins/jx-promote/pull/409/files

Reply:
A bit weird that a cli option in jx-updatebot is used in jx-promote, but then dependencies have become complex :thinking: